integration-tests/container/perf/Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM alpine:3.12 AS base
+FROM --platform=linux/amd64 alpine:3.12 AS stage-amd64
+FROM --platform=linux/ppc64le ppc64le/alpine:3.13 AS stage-ppc64le
```

Comment:
`alpine` is already a multi-arch manifest-listed image, so no need to jump through such hoops in this case.  If for whatever reason you need/want 3.13, probably just best to use it across the board.